be/src/vec/exec/join/vhash_join_node.cpp

```diff
@@ -625,14 +639,20 @@ HashJoinNode::HashJoinNode(ObjectPool* pool, const TPlanNode& tnode, const Descr
           _build_unique(_join_op == TJoinOp::LEFT_ANTI_JOIN || _join_op == TJoinOp::LEFT_SEMI_JOIN),
           _is_right_semi_anti(_join_op == TJoinOp::RIGHT_ANTI_JOIN ||
                               _join_op == TJoinOp::RIGHT_SEMI_JOIN),
-          _is_outer_join(_match_all_build || _match_all_probe) {
+          _is_outer_join(_match_all_build || _match_all_probe),
+          _output_slot_ids(tnode.hash_join_node.output_slot_ids){
     _runtime_filter_descs = tnode.runtime_filters;
     init_join_op();
 
     // avoid vector expand change block address.
     // one block can store 4g data, _build_blocks can store 128*4g data.
     // if probe data bigger than 512g, runtime filter maybe will core dump when insert data.
     _build_blocks.reserve(128);
+
+    // output slot
+    if (tnode.hash_join_node.__isset.output_slot_ids) {
```

Comment:
set it again? _output_slot_ids has been set in initialize list.

Reply:
I forget to remove it ~